raiden/api/rest.py

```diff
@@ -94,88 +105,135 @@ class RestAPI(object):
     It will provide the additional, neccessary RESTful logic and
     the proper JSON-encoding of the Objects provided by the RaidenAPI
     """
+    version = 1
 
     def __init__(self, raiden_api):
         self.raiden_api = raiden_api
         self.channel_schema = ChannelSchema()
         self.channel_list_schema = ChannelListSchema()
         self.events_list_schema = EventsListSchema()
+        self.tokens_list_schema = TokensListSchema()
+        self.partner_per_token_list_schema = PartnersPerTokenListSchema()
 
-    def open(self, partner_address, asset_address, deposit=None):
-
-        api_result = self.raiden_api.open(
-            asset_address,
+    def open(self, partner_address, token_address, settle_timeout, balance=None):
+        raiden_service_result = self.raiden_api.open(
+            token_address,
             partner_address,
+            settle_timeout
         )
 
-        if deposit:
+        if balance:
             # make initial deposit
-            api_result = self.raiden_api.deposit(
-                asset_address,
+            raiden_service_result = self.raiden_api.deposit(
+                token_address,
                 partner_address,
-                deposit
+                balance
             )
 
-        result = self.channel_schema.dumps(api_result)
+        result = self.channel_schema.dumps(raiden_service_result)
         return result
 
-    def deposit(self, asset_address, partner_address, amount):
+    def deposit(self, token_address, partner_address, amount):
 
-        api_result = self.raiden_api.deposit(
-            asset_address,
+        raiden_service_result = self.raiden_api.deposit(
+            token_address,
             partner_address,
             amount
         )
 
-        result = self.channel_schema.dumps(api_result)
+        result = self.channel_schema.dumps(raiden_service_result)
         return result
 
-    def close(self, asset_address, partner_address):
+    def close(self, token_address, partner_address):
 
-        api_result = self.raiden_api.close(
-            asset_address,
+        raiden_service_result = self.raiden_api.close(
+            token_address,
             partner_address
         )
 
-        result = self.channel_schema.dumps(api_result)
+        result = self.channel_schema.dumps(raiden_service_result)
         return result
 
-    def get_channel_list(self, asset_address=None, partner_address=None):
-        api_result = self.raiden_api.get_channel_list(asset_address, partner_address)
-        assert isinstance(api_result, list)
+    def get_channel_list(self, token_address=None, partner_address=None):
+        raiden_service_result = self.raiden_api.get_channel_list(token_address, partner_address)
+        assert isinstance(raiden_service_result, list)
 
         # wrap in ChannelList:
```

Comment:
Don't think we need these `wrap in` comments, this is precisely what the code does.